src/net/http.rs

```diff
@@ -1477,7 +1488,8 @@ impl HttpRequestType {
                 format!("/v2/contracts/source/{}/{}", contract_addr, contract_name.as_str()),
             HttpRequestType::CallReadOnlyFunction(_, contract_addr, contract_name, _, func_name, ..) => {
                 format!("/v2/contracts/call-read/{}/{}/{}", contract_addr, contract_name.as_str(), func_name.as_str())
-            }
+            },
+            HttpRequestType::OptionsPreflight(_md) => "/*".to_string(),
```

Comment:
This isn't a valid path.

Reply:
Think this should probably be some kind of panic? Basically, the OptionsPreflight RequestType shouldn't ever be used to try and create and send a request.

Reply:
I've tried to include the path in OptionsPreflight to be able to reconstruct a request. I'm definitely not sure if this is the desired style here.